ledger/sandbox/src/test/lib/scala/platform/sandboxnext/OCSPResponderFixture.scala

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandboxnext
+
+import com.daml.bazeltools.BazelRunfiles
+import com.daml.ledger.api.testing.utils.AkkaBeforeAndAfterAll
+import com.daml.ledger.api.testing.utils.OwnedResource
+import com.daml.ledger.resources.Resource
+import com.daml.ledger.resources.ResourceContext
+import com.daml.ledger.resources.ResourceOwner
+import com.daml.timer.RetryStrategy
+import org.scalatest.Suite
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.sys.process.Process
+
+trait OCSPResponderFixture extends AkkaBeforeAndAfterAll { this: Suite =>
+  private val ec: ExecutionContext = system.dispatcher
+
+  private val RESPONDER_HOST: String = "127.0.0.1"
```

Comment:
Please follow the Scala style guide for naming constants:
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods